bricksrc/location.py

```diff
@@ -4,42 +4,20 @@
 Location class structure
 """
 location_subclasses = {
-    "Building": {
-        "tags": [TAG.Site],
-    },
-    "Floor": {
-        "tags": [TAG.Floor, TAG.Location],
-    },
-    "Roof": {
-        "tags": [TAG.Roof, TAG.Location],
-    },
-    "Basement": {
-        "tags": [TAG.Basement, TAG.Location],
-    },
-    "Outside": {
-        "tags": [TAG.Outside, TAG.Location],
-    },
-    "City": {
-        "tags": [TAG.City, TAG.Location],
-    },
-    "Wing": {
-        "tags": [TAG.Wing, TAG.Location],
-    },
-    "Space": {
-        "tags": [TAG.Space, TAG.Location],
-    },
+    "Building": {"tags": [TAG.Site],},
+    "Floor": {"tags": [TAG.Floor, TAG.Location],},
+    "Roof": {"tags": [TAG.Roof, TAG.Location],},
+    "Basement": {"tags": [TAG.Basement, TAG.Location],},
+    "Outside": {"tags": [TAG.Outside, TAG.Location],},
+    "City": {"tags": [TAG.City, TAG.Location],},
+    "Wing": {"tags": [TAG.Wing, TAG.Location],},
+    "Space": {"tags": [TAG.Space, TAG.Location],},
     "Zone": {
         "tags": [TAG.Zone, TAG.Location],
         "subclasses": {
-            "HVAC_Zone": {
-                "tags": [TAG.HVAC, TAG.Zone, TAG.Location],
-            },
-            "Lighting_Zone": {
-                "tags": [TAG.Lighting, TAG.Zone, TAG.Location],
-            },
-            "Fire_Zone": {
-                "tags": [TAG.Fire, TAG.Zone, TAG.Location],
-            },
+            "HVAC_Zone": {"tags": [TAG.HVAC, TAG.Zone, TAG.Location],},
```

Comment:
Why does this have a different format than the others? I expect it to be like 
```python
                    "Cold_Box": {
                        "tags": [
                            TAG.Cold,
                            TAG.Box,
                            TAG.Laboratory,
                            TAG.Room,
                            TAG.Location,
                        ],
                    },
```

Maybe it is because of the different length of the line, but is there a way to force such things with Black?

Reply:
Second thought: I think it's fine as the formatting is automated anyway. But let me know if there is any other opinion.

Reply:
The whole point is we are relying on automated formatting :). Looks good to me!